src/esp/gfx/CubeMap.cpp

```diff
@@ -64,12 +67,44 @@ void textureTypeSanityCheck(CubeMap::Flags& flag,
       return;
       break;
 
+    case CubeMap::TextureType::VarianceShadowMap:
+      CORRADE_ASSERT(flag & CubeMap::Flag::VarianceShadowMapTexture,
+                     functionNameStr.c_str()
+                         << "instance was not created with variance shadow map"
+                            "texture output enabled.", );
+      return;
+      break;
+
     case CubeMap::TextureType::Count:
       break;
   }
   CORRADE_INTERNAL_ASSERT_UNREACHABLE();
 }
 
+/** @brief do a couple of sanity checks based on mipLevel value */
+void mipLevelSanityCheck(const std::string& msgPrefix,
```

Comment:
Same as above, maybe this instead?

```suggestion
void mipLevelSanityCheck(const char* msgPrefix,
```